_scripts/templates/Go/makefile

```diff
@@ -8,9 +8,9 @@ SOURCES = $(GENERATED) Test.go
 default: classes
 classes: setup $(SOURCES) program
 setup:
-	export | grep GO
-	go env
-	go version
+	-export | grep GO
+	-go env
+	-go version
```

Comment:
By default, "make" will stop when a command reports an error. It is possible that one does not have any "GO" environmental variables set, so the "grep" may fail. I put the '-' sign on all these lines so that it fails on the "main" build step.